README.md

```diff
@@ -1 +1,6 @@
-# Hello-Microverse-
\ No newline at end of file
+Major Languages used are 
+Html
+Frmeworks used are 
+Css
+Techonolegies used
+Git and github
```

Comment:
- [ ] What do you think about having a standard README file? It is what everyone likes, I know you aren't an exception 😄. Kindly follow this [link](https://github.com/microverseinc/readme-template) to have a copy of your own. Make sure you edit it to fit your project.